public/javascripts/tab-forms.js

```diff
@@ -0,0 +1,17 @@
+function tabForm(evt, formName) {
+
+    var i, tabcontent, tablinks;
+
+    tabcontent = document.getElementsByClassName("tabcontent");
+    for (i = 0; i < tabcontent.length; i++) {
+        tabcontent[i].style.display = "none";
+    }
```

Comment:
This is totally fine, but you could probably simplify it like..

```
var tabcontent = document.getElementsByClassName('tabcontent');

tabcontent.forEach(function(tc) {
  tc.style.display = 'none';
});
```

`Array.prototype.forEach` is available in pretty much all browsers - IE8 and lower doesn't have it, but if [Microsoft doesn't support IE < 11](https://www.microsoft.com/en-us/WindowsForBusiness/End-of-IE-support), why should we?